.changeset/twelve-bobcats-collect.md

```diff
@@ -0,0 +1,5 @@
+---
+"@trigger.dev/cli": minor
```

Comment:
This needs to be a "patch" change. Just change this file (need need to run `pnpm run changeset:add` again

Reply:
@ericallam kindly take a look.

Reply:
Okay now you just need to delete this file cause you've already got another changeset file

Reply:
Ah shoot! So after deleting it... I won't need to `pnpm changeset:add` again, yeah?

Reply:
Nope, just delete this `.changeset/twelve-bobcats-collect.md` file and the PR will be good to go

Reply:
Awesome! I'll do that now.

Reply:
Pushed! 🚀